src/Components/Nav/NavList/NavList.js

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from 'react';
+import NavCategoryList from './NavCategoryList/NavCategoryList'
+import NavProductList from './NavProductList/NavProductList';
+import { categories } from './PropertyData'
+import './NavList.scss'
+
+
+
+class NavList extends Component {
+  constructor() {
+    super();
+    this.state = {
+      categories: [],
+      categoryIdx: 0,
+      subCategoryIdx: 0,
+      isCategoryBoxVisible: false,
+      isProductBoxVisible: false,
+    };
+  }
+
+  componentDidMount() {
+    this.setState({
+      categories : categories,
```

Comment:
객체에서 key와 value가 동일하다면 하나로 줄일 수 있습니다!
```suggestion
      categories,
```